neuralop/training/torch_setup.py

```diff
@@ -19,6 +21,10 @@ def setup(config):
         device : torch.device
         is_logger : bool
     """
+    # We use Torchrun for spawning MP jobs, and torchrun auto-initializes env vars. 
+    # Check for override in case config option conflicts with the fact that we are inside a torch distributed elastic run.
+    if os.getenv("LOCAL_RANK") is not None:
+        config.distributed.use_distributed = True 
```

Comment:
Is that robust? Don't we want users to specify this? Seems risky to override users config without warning.